### PR TITLE
Add API to fetch a string

### PR DESCRIPTION
**Pull request by Gymnasiast**
You can blame @Manticore-007 for this...

### CODE REVIEW

File: distribution/openrct2.d.ts [hunk 1]
```diff
@@ -240,6 +240,8 @@ declare global {
          */
         getIcon(iconName: IconName): number;
 
+        getStringById(stringId: number): string;
```
Comment (tupaschoal) [inline distribution/openrct2.d.ts:243]:
```suggestion
        /**
         * Gets the in-game string according to the current language
         * @param stringId The string number id as defined on src/openrct2/localisation/StringIds.h.
         */
        getStringById(stringId: number): string;
```

**Review by tupaschoal — APPROVED** [review]

**Comment by IntelOrca** [comment]
Should there be a disclaimer? That a string ID is not guaranteed to return the same result or even be the same area between builds? With icons, names are now used.

**Comment by Gymnasiast** [comment]
> or even be the same area 

What do you mean, exactly? Unlike with G2 ids, removing a string does not change the IDs of the rest.

**Comment by IntelOrca** [comment]
> > or even be the same area
> 
> What do you mean, exactly? Unlike with G2 ids, removing a string does not change the IDs of the rest.

I am just wondering whether it is a good idea to lock us into the current string ID model.

**Comment by Basssiiie** [comment]
> I am just wondering whether it is a good idea to lock us into the current string ID model.

Isn't this system already locked considering there are like 20 localisations? It wouldn't be realistic to ask all of them to re-localise the game a second time using a different system.

**Comment by Gymnasiast** [comment]
> > I am just wondering whether it is a good idea to lock us into the current string ID model.
> 
> Isn't this system already locked considering there are like 20 localisations? It wouldn't be realistic to ask all of them to re-localise the game a second time using a different system.
> 
> Though if there are actually plans for a different system, then yes that is very important to consider before merging this.

There are, though the question is how long that’s going to take. Nevertheless, exposing this to plug-in _would_ mean quite a restriction on moving it.  So I guess it might be better not to merge this.

**Pull request closed without merging**